app_flutter/lib/service/appengine_cocoon.dart

```diff
@@ -253,7 +253,8 @@ class AppEngineCocoonService implements CocoonService {
       ..requiredCapabilities.add(objectRequiredCapabilities.toString())
       ..reservedForAgentId = taskData['ReservedForAgentID']
       ..stageName = taskData['StageName']
-      ..status = taskData['Status'];
+      ..status = taskData['Status']
+      ..isTestFlaky = taskData['TestFlaky'];
```

Comment:
```suggestion
      ..isTestFlaky = taskData['TestFlaky'] ?? false;
```

To handle release branches

Reply:
Release tasks should be the same as master ones, with default value `false`.
But added to incorporate old tasks with `null` value.